mount/mount.go

```diff
@@ -22,3 +22,19 @@ func MountAll(mounts []Mount, target string) error {
 	}
 	return nil
 }
+
+// UnmountN tries to unmount the given mount point nr times, which is
+// useful for undoing a stack of mounts on the same mount
+// point. Returns the first error encountered, but always attempts the
+// full nr umounts.
+func UnmountN(mount string, flags, nr int) error {
+	var err error
+	for i := 0; i < nr; i++ {
```

Comment:
I figured there was no harm in trying to do all of them, but yes it is likely not to do much in practice after the first failure and will certainly not unmount the full stack if one of them fails.